test/jit/test_backends.py

```diff
@@ -465,3 +465,126 @@ def test_save_load(self):
     @skipIfRocm
     def test_errors(self):
         self.selective_lowering_test.test_errors()
+
+
+# Unit Tests for backend with compiler
```

Comment:
Add more comments here: 
"The reason to add it as a separate test case, than reusing the existing TestBackends, is these two cases cover different aspects. The actually backend implementation in this test is different. It has a simple demo compiler to test the end-to-end flow. However, this test cannot cover the selective_lowering for now, which is covered in TestBackends."